extensions/amp-story/1.0/amp-story.js

```diff
@@ -650,16 +651,48 @@ export class AmpStory extends AMP.BaseElement {
     });
   }
 
+
   /**
-   * Ensures publishers using amp-consent use amp-story-consent.
+   * Handles story consent.
    * @private
    */
-  validateConsent_() {
+  handleConsent_() {
```

Comment:
nit: "handle consent" sounds to me like the user has provided consent and we are handling their response; can we make it more clear in the naming and/or comment that this is not the case?

Reply:
Oh right! I changed it for ``handleConsentExtension_``, let me know if you have a better idea...